projects/packages/my-jetpack/_inc/components/product-card/index.tsx

```diff
@@ -24,51 +25,36 @@ export const PRODUCT_STATUSES_LABELS = {
 	[ PRODUCT_STATUSES.CAN_UPGRADE ]: __( 'Active', 'jetpack-my-jetpack' ),
 };
 
-// SecondaryButton component
-const SecondaryButton = props => {
-	const {
-		shouldShowButton = () => true,
-		positionFirst,
-		...buttonProps
-	} = {
-		size: 'small',
-		variant: 'secondary',
-		weight: 'regular',
-		label: __( 'Learn more', 'jetpack-my-jetpack' ),
-		...props,
-	};
-
-	if ( ! shouldShowButton() ) {
-		return false;
-	}
-
-	return <Button { ...buttonProps }>{ buttonProps.label }</Button>;
-};
-
-SecondaryButton.propTypes = {
-	href: PropTypes.string,
-	size: PropTypes.oneOf( [ 'normal', 'small' ] ),
-	variant: PropTypes.oneOf( [ 'primary', 'secondary', 'link', 'tertiary' ] ),
-	weight: PropTypes.oneOf( [ 'bold', 'regular' ] ),
-	label: PropTypes.string,
-	shouldShowButton: PropTypes.func,
-	onClick: PropTypes.func,
-	positionFirst: PropTypes.bool,
-	isExternalLink: PropTypes.bool,
-	icon: PropTypes.node,
-	iconSize: PropTypes.number,
-	disabled: PropTypes.bool,
-	isLoading: PropTypes.bool,
-	className: PropTypes.string,
+export type ProductCardProps = {
+	children?: ReactNode;
+	name: string;
+	Description: FC;
+	admin: boolean;
+	recommendation?: boolean;
+	isFetching?: boolean;
+	isDataLoading?: boolean;
+	isInstallingStandalone?: boolean;
+	isManageDisabled?: boolean;
+	onActivate?: MouseEventHandler< HTMLButtonElement >;
+	slug: string;
+	additionalActions?: unknown[];
```

Comment:
I believe `additionalActions` here are really just an array of Button component props.  You can see an example in the Backup card where additional actions is passed along. I don't think we can import the Button component props, but we could probably use `SecondaryButtonProps` which is similar to the Button props but with a couple additional added props. What do you think? Do you see any problem using `SecondaryButtonProps[]` to define the `additionalActions` prop?

```suggestion
	additionalActions?: SecondaryButtonProps[];
```

Reply:
It's a good idea, thanks!